core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnPredictionOp.scala

```diff
@@ -0,0 +1,39 @@
+package edu.uci.ics.texera.workflow.operators.sklearn
+
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+class SklearnPredictionOp extends PythonOperatorDescriptor {
+  override def generatePythonCode(): String =
+    s"""from pytexera import *
+       |class ProcessTupleOperator(UDFOperatorV2):
+       |    @overrides
+       |    def process_tuple(self, tuple_: Tuple, port: int) -> Iterator[Optional[TupleLike]]:
+       |        if port == 0:
+       |            self.model = tuple_["model"]
```

Comment:
can user choose the attribute name on the UI? right now it is hard-coded as "model".

Reply:
Fixed.